aix/AixProcess.h

```diff
@@ -0,0 +1,55 @@
+/* Do not edit this file. It was automatically generated. */
+
+#ifndef HEADER_AixProcess
+#define HEADER_AixProcess
+/*
+htop - AixProcess.h
+(C) 2015 Hisham H. Muhammad
+(C) 2018 Calvin Buckley
+Released under the GNU GPL, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "Settings.h"
+#include <sys/corralid.h> // cid_t 
+
+#define Process_delete AixProcess_delete
+
+typedef enum AixProcessFields {
+   // Add platform-specific fields here, with ids >= 100
+   WPAR_ID = 100,
+   LAST_PROCESSFIELD = 101,
+} AixProcessField;
+
+typedef struct AixProcess_ {
+   Process super;
+   int kernel;
+   cid_t cid; // WPAR ID
+   unsigned long long int stime, utime; // System/User time (stored sep for calculations)
```

Comment:
I don't recommend defining multiple variables on the same line. Especially in a structure.